UKHomeOffice/cerberus-service
# COP-5754 Refresh task list

## Description

- TaskList should refresh every 60 seconds with the latest results from camunda
- The user should remain on the same page (pagination wise) when the list refreshes
- The tasks on this page may change, if tasks are sorted by newest first

## To Test

- Run locally
- Open network tab
- Open the /tasks page (default landing page)
- Wait 60sec
> You should see another instance of the API call

To test pagination

- update code to set limit to at least 5 (forces pagination)
- run locally
- open network tab
- open the /tasks page
- go to page 2
- wait 60 sec
- You should see another instance of the API call
> The URL & pagination should still show you are on page 2

_Note: you can set the interval to 2000 (2sec) if you don't want to wait for 60sec!_

## Developer Checklist

\* Required
